### PR TITLE
VATINValidator should also be able to handle Python Strings

### PR DESCRIPTION
**Pull request by rm--**
See #32 for details.

### CODE REVIEW

**Comment by codingjoe** [comment]
Hi @rm--, tho I don't get the usecase, why you'd pass a string, I don't mind this change and appreciate you being invested and contributing. Thanks!

A couple of small things:
- [ ] Can you please add a test, that check the new signature and leave the others as is.
- [ ] Can you drop the function and add this behavior to the Validators `__init__` method
- [ ] Can you `trim` the string first?

**Comment by codingjoe** [comment]
Ok I read the ticket and the PR in the wrong order, sorry.
Can you also change the documentation? And provide a test exploiting the error you described?

**Comment by rm--** [comment]
Hey @codingjoe,

I changed the tests. Now both signatures  (existing with and new one without VATIN object) will be tested.
The function `split_vatin` could be dropped but not handled in the `__init__` method. I suggest to keep `split_vatin` as an additional method because it is more clear and there is a doc string to explain the reason. I used `split` to trim the values. What do you think?

The documentation is updated regarding the typo. Do you have other changes in mind? 
The (new) tests without the fix break the existing code. You can run them if you checkout commit `07b4f3b`.

File: vies/validators.py [hunk 1]
```diff
@@ -16,7 +18,13 @@ def __init__(self, verify=True, validate=False):
...
 
     def __call__(self, value):
+        if not isinstance(value, VATIN):
+            value = VATIN(*self.split_vatin(value))
         if self.verify:
             value.verify()
         if self.validate:
             value.validate()
+
+    def split_vatin(self, value):
```
Comment (codingjoe) [inline vies/validators.py]: Can you maybe put this in a factory method on the VATIN class itself.
e.G. `VATIN.from_str('DE123456789')`

File: vies/validators.py [hunk 2]
```diff
@@ -16,7 +18,13 @@ def __init__(self, verify=True, validate=False):
         self.validate = validate
 
     def __call__(self, value):
+        if not isinstance(value, VATIN):
```
Comment (codingjoe) [inline vies/validators.py]: please add test, that the value is infact a `text_type`

File: tests/test_validators.py [hunk 3]
```diff
@@ -8,38 +8,68 @@
...
+        validator = VATINValidator()
+        validator(valid_vatin)
+
+        validator = VATINValidator(verify=False, validate=True)
+        validator(valid_vatin)
+
+        validator = VATINValidator(verify=True, validate=True)
+        validator(valid_vatin)
+
+    def test_invalid(self):
```
Comment (codingjoe) [inline tests/test_validators.py]: you can [parametrise a test](http://docs.pytest.org/en/latest/parametrize.html) so it runs with both a string and and VATIN.
I'd prefer tho, if you just add a test for the new method. And leave the rest as is.

**Comment by codingjoe** [comment]
Hi @rm-- this looks really good. I left you a couple more comments.
Please make sure to also add documentation for the new `VATIN.from_str` factory I encouraged you to add. This would be release in version 3.1.0 since it not only fixes a bug, but adds a new feature :)

**Comment by rm--** [comment]
I updated the code by your comments. Feel free to modify the lines if something is not in the way you like it. ;)

**Comment by codingjoe** [comment]
Released in 3.1.0

**Comment by rm--** [comment]
Thanks!

**Pull request merged**